src/app/public/Pages/homepage/homepage.component.html

```diff
@@ -0,0 +1,59 @@
+<!-- Add bulma components to the CSS page later, setting up initial pages and routing to get it over with -->
+<!-- Getting a weird bug that doubles the HTML onto the page whenever I put anything on this page - Anthony -->
+<!-- Attempting to get something like the top half of the DoorDash Website so it looks aesthetically pleasing. - Anthony -->
+<section class="hero is-medium is-info">
+  <div class="hero-body">
+    <div class="container has-text-centered">
+      <!-- Placeholder for an image here - Anthony -->
+      <h1 class="title is-1 mt-auto">
+        Go Mechanic
+      </h1><br>
+      <h2 class="subtitle is-4 mb-6">
+        <i>Home of your mobile mechanic needs!</i>
+      </h2>
+      <form>
+        <div class="control is-loading">
+          <!-- Placeholder address search for our Google Maps API we will add later to search for mechanics near you. - Anthony -->
+          <label>
+            <input class="input" type="text" placeholder="Search for mechanics near you">
+          </label>
+        </div>
+      </form>
+    </div>
+  </div>
+</section>
+
+<!-- More information about the website, below is just filler to have a mock-up of what we want it to look like and function. - Anthony -->
+<section class="section">
+  <div class="container has-text-centered">
+    <p class="subtitle">
+      Listed below are some examples of the repairs you can find here at Go Mechanic. <!-- All text is subject to change at the discretion of the group, just wanted to get some stuff so it looks presentable to an extent. - Anthony -->
+    </p>
+  </div>
+</section>
+
+<!-- Keep track of div tags and make sure to not mix and match Bulma and Angular classes, please! - Anthony -->
+<!-- Columns of the different kinds of work that you can have done on your vehicle listed below. - Anthony -->
+<!-- Someone can make it look prettier later, I'm just getting things situated for the most part. - Anthony -->
+
+<!-- The figure class lines will be replaced with a basic picture template of hypothetical prices-->
+<br>
+<div class="columns is-mobile is-gapless is-vcentered has-text-centered">
+  <div class="box column is-size-4 is-family-monospace">
+    <br>Maintenance<br>
+    <figure class="image is-3by5"><img src="https://bulma.io/images/placeholders/480x800.png" alt="Maintenance"></figure>
+  </div>
+  <div class="box column is-size-4 is-family-monospace">
+    <br>Engine Light<br>
+    <figure class="image is-3by5"><img src="https://bulma.io/images/placeholders/480x800.png" alt="Engine Light"></figure>
+  </div>
+  <div class="box column is-size-4 is-family-monospace">
+    <br>Brake Job<br>
+    <figure class="image is-3by5"><img src="https://bulma.io/images/placeholders/480x800.png" alt="Brakes"></figure>
+  </div>
+  <div class="box column is-size-4 is-family-monospace">
+    <br>More Repairs!<br>
+    <figure class="image is-3by5"><img src="https://bulma.io/images/placeholders/480x800.png" alt="More repairs!">
+    </figure>
+  </div>
+</div>
```

Comment:
This component gets changed multiple times unnecessarily. We need to have a merge that gets all the changes integrated.